metrics2/metrics2/src/main/java/io/helidon/metrics/EWMA.java

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
This class is exact duplicate of the one in `metrics` - maybe extract to common metrics?